internal/commands/inspect.go

```diff
@@ -3,40 +3,70 @@ package commands
 import (
 	"fmt"
 
+	"github.com/deislabs/cnab-go/action"
 	"github.com/docker/app/internal"
+	appstore "github.com/docker/app/internal/store"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/config"
 	"github.com/spf13/cobra"
 )
 
 type inspectOptions struct {
-	parametersOptions
-	pullOptions
+	pretty bool
 }
 
 func inspectCmd(dockerCli command.Cli) *cobra.Command {
 	var opts inspectOptions
 	cmd := &cobra.Command{
 		Use:     "inspect [APP_NAME] [OPTIONS]",
```

Comment:
@chris-crone What should we put here: `[APP_NAME]`? `[APP_IMAGE]`? `[APP_INSTALLATION]`? Both as `inspect` can be applied on installation or app image?